frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -42,9 +42,9 @@ export const eventUsageLogic = kea<
             instanceEmailAvailable,
         }),
         reportFunnelCalculated: (
-            eventCount: number,
-            actionCount: number,
-            interval: string,
+            eventCount: number | undefined,
```

Comment:
These can be formatted like `eventCount?: number`

Reply:
When I changed it a bunch of stuff didn't like that 😢